### PR TITLE
feat: add web-component

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:

Improve web component to be production ready:
- Update code inside `web-component` folder,
- Update main Readme.md
- Update github workflow for automatically release of web-component

**Related issue(s)**
See also https://github.com/asyncapi/asyncapi-react/pull/148

@gexclaude if you would like to, I invite you to check the PR, especially the changes in readme :)

### CODE REVIEW

**Comment by gexclaude** [comment]
Looks good to me - I'll be more than happy to check the README again as a user when switching from our custom build.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -16,36 +16,61 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: 13
-      - name: Install dependencies for library and playground
+      - name: Install dependencies for library. web-component and playground
```
Comment (derberg) [inline .github/workflows/release.yml:19]:
```suggestion
      - name: Install dependencies for library, web-component and playground
```

**Review by derberg — CHANGES_REQUESTED** [review]
LGTM
just 2 things:
- it should be published to `@asyncapi/web-component`
- can we rename `async-api-web-component.js` to `asyncapi-web-component.js` and any other `async-api` occurances too? 
- you need to have this https://github.com/asyncapi/html-template/blob/master/package.json#L28-L30 in package.json of webcomponent otherwise first publishing to npm will fail

**Comment by magicmatatjahu** [comment]
@derberg I propagated your suggestions to PR :)

**Review by derberg — APPROVED** [review]
LGTM
let's see if it works 😄

**Pull request merged**